### DISCUSSION TITLE
UK Met Office DataHub

### DISCUSSION THREAD

**Discussion by bunburya** [breezy-weather/breezy-weather#1384] [Q&A] [answered]
In the discussion at #923, the Met Office's DataHub was not accepted as a source on the basis that the user is required to provide their phone number to obtain an API key. Looking at it again, phone number is optional and I can confirm I obtained an API key without entering it. It seems the mandatory information is: an email address, your name, your "country of origin" and how you heard about the service.

My question is, do the facts above justify permitting the Met Office DataHub as a data source? If so, I would create a new source request (and, subject to other time commitments, would be happy to look at implementing it). But I wanted to check first, in case the information that is requested/required still renders it inappropriate as a source for Breezy.

**Comment by papjul** [comment, marked as answer]
Maybe they lowered the requirements since then. Yes, please fill in a new "Source" template issue, and I will review it again later ;)
